pygments/lexers/cplint.py

```diff
@@ -0,0 +1,54 @@
+"""
+    pygments.lexers.cplint
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the cplint language https://cplint.eu, including CP-logic, Logic Programs with Annotated Disjunctions, 
+    ProbLog and Distributional Clauses syntax
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import bygroups, inherit, words
+from pygments.lexers import PrologLexer
+from pygments.token import Text, Comment, Operator, Keyword, Name, String, \
+    Number, Punctuation
+
+__all__ = ['CplintLexer']
+
+
+class CplintLexer(PrologLexer):
+    """
+    Lexer for cplint files, https://cplint.eu, including CP-logic, Logic Programs with Annotated Disjunctions, 
+    ProbLog and Distributional Clauses syntax
+      .. versionadded:: 2.11
+    """
+    name = 'cplint'
+    aliases = ['cplint']
```

Comment:
I believe this is unneeded since the name covers you already.

Reply:
I thought so as well but in reality it is needded, otherwise the lexer is not picked up